ReactiveCocoa/UIKit/UIActivityIndicatorView.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  UIActivityIndicatorView.swift
+//  Rex
+//
+//  Created by Evgeny Kazakov on 02/07/16.
+//  Copyright © 2016 Neil Pankey. All rights reserved.
+//
+
+import ReactiveSwift
+import UIKit
+
+extension UIActivityIndicatorView {
+
+	/// Wraps an indicator's `isAnimating()` state in a bindable property.
+	/// Setting a new value to the property would call `startAnimating()` or
+	/// `stopAnimating()` depending on the value.
+	public var rac_animating: BindingTarget<Bool> {
```

Comment:
I think `rac` makes sense for now. It's easily pronounceable and fits with the history of the project.

We can consider an operator in the future, but that could be an additive change.